docs/user-guide/advanced/Pandas_API.ipynb

```diff
@@ -1813,6 +1917,136 @@
     "df.astype({'c4':kx.SymbolVector, 'c5':kx.SymbolVector})"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "0f8813a0",
+   "metadata": {},
+   "source": [
+    "### Table.add_prefix()\n",
+    "\n",
+    "```\n",
+    "Table.add_prefix(columns)\n",
+    "```\n",
+    "\n",
+    "Rename columns adding a prefix in a table and return the resulting Table object.\n",
+    "\n",
+    "**Parameters:**\n",
+    "\n",
+    "| Name    | Type            | Description                                                         | Default    |\n",
+    "| :-----: | :-------------: | :------------------------------------------------------------------ | :--------: |\n",
+    "| prefix    | str  | The string that will be concatenated with the name of the columns  | _required_ |\n",
+    "| axis    | int                    | Axis to add prefix on.     | 0          |\n",
+    "\n",
+    "**Returns:**\n",
+    "\n",
+    "| Type  | Description                                                        |\n",
+    "| :---: | :----------------------------------------------------------------- |\n",
+    "| Table | A table with the given column(s) renamed adding a prefix.                          |"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9186ed86",
+   "metadata": {},
+   "source": [
+    "**Examples:**\n",
+    "\n",
+    "The initial table to which a prefix will be added to its columns"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5f20131b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tab.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "73c2b08f",
+   "metadata": {},
+   "source": [
+    "Add \"col_\" to table columns:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "926c8295",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tab.add_prefix(prefix=\"col_\").head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0a4abc8c",
+   "metadata": {},
+   "source": [
+    "### Table.add_sufix()\n",
```

Comment:
In the notebook it should be `suffix` rather than `sufix` which matches the implementation

Reply:
Absolutely right (sorry for the typos). Already solved.